### PR TITLE
Extract `HcbCode#memo` to a concern

### PR DESCRIPTION
**Pull request by cjdenio**
I've noticed the HcbCode model getting crowded with quite a few methods for generating transaction memos. This is an attempt to clean it up a little!

### CODE REVIEW

File: app/models/hcb_code.rb [hunk 1]
```diff
@@ -26,6 +26,8 @@ class HcbCode < ApplicationRecord
   include Commentable
   include Receiptable
 
+  include HcbCode::Memo
```
Comment (northeastprince) [inline app/models/hcb_code.rb]:
```suggestion
  include Memo
```

File: app/models/hcb_code/memo.rb [hunk 2]
```diff
@@ -0,0 +1,91 @@
+# frozen_string_literal: true
+
+class HcbCode
+  module Memo
```
Comment (northeastprince) [inline app/models/hcb_code/memo.rb]:
```suggestion
module HcbCode::Memo
```

Comment (cjdenio) [reply app/models/hcb_code/memo.rb]: I think we have a Rubocop rule that enforces the first style: https://docs.rubocop.org/rubocop/cops_style.html#styleclassandmodulechildren

Comment (northeastprince) [reply app/models/hcb_code/memo.rb]: I prefer to one-line namespaced classes/modules (unless the full definition is huge), but it's not a big deal.

**Review by garyhtou — APPROVED** [review]
AMAZING!! i really appreciate this one!

assuming that we're just moving the methods into the concern with no other changes, we're good to go!

**Comment by garyhtou** [comment]
Next up might be a `LinkedObjects` concern for stuff like `ach_transfer?` and `ach_transfer`

**Pull request merged**